frontend/app/controllers/Redirects.scala

```diff
@@ -0,0 +1,14 @@
+package controllers
+
+import play.api.mvc.Controller
+
+trait Redirects extends Controller {
+
+  // Subscriber Offer
+  def subscriberOffer = CachedAction {
```

Comment:
Yeah, good point. I was just reusing the original route name, but it could just be renamed.